src/index.js

```diff
@@ -1,4 +1,9 @@
-import { obtenerNombreProducto } from "./productos";
+//import { obtenerNombreProducto } from "./productos";
+import { obtenerBotonComprar } from "./productos";
+import { agregarProducto1 } from "./carrito";
+import { agregarProducto2 } from "./carrito";
+import { agregarProducto3 } from "./carrito";
+import { agregarProducto4 } from "./carrito";
```

Comment:
cuando hagas el refactor anterior, solo deberias importar una sola funcion ->
```suggestion
import { agregarProducto } from "./carrito";
```